specification/security/resource-manager/Microsoft.Security/preview/2024-08-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1785 @@
+{
+  "swagger": "2.0",
```

Comment:
>"swagger": "2.0", [](http://example.com/codeflow?start=2&length=17)

[ARMBlockingComment] All types from previous versions need to be carried forward to the new API version

Reply:
can you please elaborate on this? all types from this file are carried to the new API version.

Reply:
Confirmed with @rkmanda that the original comment no longer applies since you've reset the commits so we can see the proper diffs for this PR now.